kcidb/test_oo.py

```diff
@@ -0,0 +1,454 @@
+"""kcidb.oo test module"""
+
+import kcidb
+from kcidb import db
+from kcidb import oo
+
+
+class KCIDB00ClientTestCase(kcidb.unittest.TestCase):
+    """Test case for kcidb.oo module"""
+
+    def setUp(self):
+        """Database setup and oo client creation"""
+        self.source = db.Client('sqlite::memory:')
```

Comment:
We don't really use `self.source` anywhere, but in `initialize_and_load_data`, so let's not store it in the instance, and instead just pass it to the latter explicitly.